### DISCUSSION TITLE
Using a local repository

### DISCUSSION THREAD

**Discussion by Kidaev** [Vinzent03/obsidian-git#719] [Q&A] [answered]
Can I use this plugin for the repository located on my PC? 
For example, I have no desire to push changes to a remote repository or I do not have access to the network.

**Comment by Vinzent03** [comment, marked as answer]
yeah sure works just fine. You just have to configure everything correctly in the settings. Basically just disable every push and pull setting.

**Reply by Kidaev** [reply]
Thanks!!! ❤️‍🔥
